libraries.cmake/seqan.cmake

```diff
@@ -14,6 +14,11 @@ MACRO( OPENMS_CONTRIB_BUILD_SEQAN )
   endif()
   OPENMS_SMARTEXTRACT(ZIP_ARGS ARCHIVE_SEQAN "SEQAN" "include/seqan/version.h")
 
+  # https://github.com/OpenMS/contrib/issues/81
```

Comment:
Could you annotate: `required for compiler bug in GCC 4-5.X` or similar. So we know directly when to remove the patch later on (when our support for these compilers runs out)